airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.78.7
```

Comment:
This is updated automatically by the bump version script that is part of the GitHub action. I'm not sure what happens when we try to manage this manually so I think it would be better to revert that line

Reply:
Sounds good, thanks for the clarification! Just to confirm, should I leave in the changelog message itself, or will it also auto-create one based on some value like the PR title?

Reply:
It'll get populated based on the value you enter on the last field of [this GitHub action](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml)